funding/manager.go

```diff
@@ -3115,11 +3114,18 @@ func (f *Manager) addToRouterGraph(completeChan *channeldb.OpenChannel,
 
 	fwdMinHTLC, fwdMaxHTLC := f.extractAnnounceParams(completeChan)
 
+	// A channel is not announced to the network if it is explicitly set
+	// or when a peerAlias is available which signals a zeroconf channel.
+	announcedChan := completeChan.ChannelFlags&lnwire.FFAnnounceChannel != 0
+	if peerAlias != nil {
+		announcedChan = false
+	}
```

Comment:
Was doing it because of the readability of the code, the 80 character limit would lead to something like this:

```
announcedChan := completeChan.ChannelFlags&
		lnwire.FFAnnounceChannel != 0 && peerAlias == nil
```

But ok to change if that's better?

Reply:
the fewer times we reassign a variable the better imo. If you break it up put the second boolean condition on a new line rather than breaking the mask operation across lines.